Book-A-Desk.Api/Api.Models.fs

```diff
@@ -9,7 +10,7 @@ type UserReference =
 
 type OfficeReference =
     {
-        Id: string
+        Id: OfficeId
```

Comment:
As this is an API model and the specification says that the Id is a string this is wrong here. Also there should be no namespace reference to the domain models required here.